index.html

```diff
@@ -4,69 +4,65 @@
     <meta charset="UTF-8" />
     <title>🏎️ 자동차 경주 게임</title>
     <link rel="stylesheet" href="./src/css/index.css" />
+    <script type="module" src="./src/js/index.js"></script>
   </head>
   <body>
     <div id="app">
       <div class="d-flex justify-center mt-5">
         <div>
-          <section>
+          <section class="title-container">
             <h1 class="text-center">🏎️ 자동차 경주 게임</h1>
+          </section>
+          <section class="car-name-input-container">
             <p>
               5자 이하의 자동차 이름을 콤마로 구분하여 입력해주세요. <br />
               예시) EAST, WEST, SOUTH, NORTH
             </p>
-          </section>
-          <section>
             <div class="d-flex">
-              <input type="text" class="w-100 mr-2" placeholder="자동차 이름" />
-              <button type="button" class="btn btn-cyan">확인</button>
+              <input
+                type="text"
+                class="car-name-input-container__input w-100 mr-2"
+                placeholder="자동차 이름"
+              />
+              <button
+                type="button"
+                class="car-name-input-container__button btn btn-cyan"
+              >
+                확인
+              </button>
             </div>
           </section>
-          <section class="mt-5">
+          <section class="lap-count-input-container mt-5" hidden>
```

Comment:
`display: flex`를 지정한 요소는 hidden 특성이 존재하더라도 화면에 보이게 됩니다! visibile은 `display` 속성을 제어하면 어떨까요?

Reply:
hidden attribute를 이번에 처음 알게 되어, 너무 간편하다고 생각하여 깊이 찾아보지 않고 성급하게 적용하였습니다..
말씀하신 사항이 이번 구현시에는 발생하지 않아서, 말씀해주지 않았으면 절대로 몰랐을 것 같습니다. 감샇바니다.

html spec을 찾아보니, 일반적으로 브라우져에서는 이를 CSS를 적용하여 구현하므로, HTML Element의 display 속성을 CSS에서 명시적으로 지정한 경우에는, hidden 속성이 있더라도 이를 override 하게 되는 것으로 이해하였습니다.

> Because this attribute is typically implemented using CSS, it's also possible to override it using CSS. For instance, a rule that applies 'display: block' to all elements will cancel the effects of the hidden attribute. Authors therefore have to take care when writing their style sheets to make sure that the attribute is still styled as expected.


CSS에서 `--hidden` 클래스를 정의하여 이를 통해 구현하도록 수정하였습니다.

#### 추후 정리용 참고 Reference) 
- https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden
- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

Reply:
관련 커밋입니다:  https://github.com/woowacourse/javascript-racingcar/pull/32/commits/ff4b7d3d7c8b28021eb78a1979ac8ed68bb2f9d1

Reply:
정리 끝판왕이시네요 👏 👏 👏 👏 👏 👏 👏